paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/CustomerSheetTest.kt

```diff
@@ -40,6 +41,7 @@ internal class CustomerSheetTest {
 
     @get:Rule
     val chain: RuleChain = RuleChain.emptyRuleChain()
```

Comment:
Should we update this to use `TestRules.create()` here and in the other updated test files?

Reply:
TestRules is a playground specific thing. This code is in the paymentsheet module, so can't share it, and we wouldn't want to for this exact thing. But we could probably clean this up to use something similar to what we do for `TestRules`. I'll follow up.